pymysql/tests/test_issues.py

```diff
@@ -381,3 +383,30 @@ def test_issue_175(self):
                     warnings.filterwarnings("ignore")
                     cur.execute('drop table if exists test_field_count')
 
+    def test_pull_request_for_fix_tuple_and_unicode_encoding_issue(self):
+        """ Encoding error appears when iterable is passed in query args list.
```

Comment:
Thnx for feedback.
I'll apply your comments on next week.

Sent from my HTC

----- Reply message -----
From: "INADA Naoki" <notifications@github.com>
To: "PyMySQL/PyMySQL" <PyMySQL@noreply.github.com>
Cc: "Zhiltsov, Sergey" <sergey.zhiltsov@sigmaukraine.com>
Subject: [PyMySQL] Encoding error appears when iterable is passed in query args list. (#321)
Date: Sat, May 30, 2015 00:52


In pymysql/tests/test_issues.py<https://github.com/PyMySQL/PyMySQL/pull/321#discussion_r31368045>:

> @@ -381,3 +383,30 @@ def test_issue_175(self):
>                      warnings.filterwarnings("ignore")
>                      cur.execute('drop table if exists test_field_count')
>
> +    def test_pull_request_for_fix_tuple_and_unicode_encoding_issue(self):
> +        """ Encoding error appears when iterable is passed in query args list.


And could you add link to this issue in docstring?

-
Reply to this email directly or view it on GitHub<https://github.com/PyMySQL/PyMySQL/pull/321/files#r31368045>.